hasher-matcher-actioner/hmalib/common/message_models.py

```diff
@@ -84,19 +85,24 @@ class ActionMessage(MatchMessage):
     action_label: ActionLabel = ActionLabel("UnspecifiedAction")
     action_rules: t.List[ActionRule] = field(default_factory=list)
 
+    # from content
+    additional_fields: t.List[str] = field(default_factory=list)
+
     @classmethod
-    def from_match_message_action_label_and_action_rules(
+    def from_match_message_action_label_action_rules_and_additional_fields(
```

Comment:
nit/temporarily ignorable: this has a smell to me (beyond nit on the name), taking an object you inherit from directly and accessing its fields manually to initialize feels off. Flagging as this is making me question the interface/abstraction but that is likely beyond the scope of this PR.

Reply:
Yep I agree. Open to refactor later